src/main/java/games/strategy/engine/lobby/server/userDB/DbUser.java

```diff
@@ -1,42 +1,113 @@
 package games.strategy.engine.lobby.server.userDB;
 
-import java.util.Date;
+import com.google.common.base.Preconditions;
 
+import games.strategy.engine.framework.startup.ui.InGameLobbyWatcher;
+import games.strategy.util.Util;
+
+/**
+ * Value object representing data needed to create a user in database.
+ */
 public class DbUser {
-  private final String name;
-  private final String email;
-  private final boolean isAdmin;
-  private final Date lastLogin;
-  private final Date joined;
+  public final String name;
+  public final String email;
+  public final boolean admin;
```

Comment:
Setters would make these fields writable, that is not desirable.
We're following a pretty well trodden pattern for value objects here with an all arg constructor and public final member variables. Not having getters helps enforce the notion that the values are immutable. Ultimately getters would be okay, but they do not add anything.